src/pages/Profiles.vue

```diff
@@ -115,7 +115,7 @@ export default class Profiles extends ProfilesMixin {
         const settings = await this.$store.getters.settings;
         await settings.load();
 
-        const lastProfileName = await this.$store.dispatch('profile/loadLastSelectedProfile');
+        const lastProfileName = "Default";
```

Comment:
What's the rationale behind this change? It just plain breaks the existing behaviour, and I don't think we want to remove it (and if we did then there would be more cleanup to remove the dead code).